cobra/__init__.py

```diff
@@ -194,11 +195,11 @@ def __init__(self, socket, sflags=0):
                 raise Exception('Missing "msgpack" python module ( http://visi.kenshoto.com/viki/Msgpack )')
 
             def msgpackloads(b):
-                logger.debug("<< %r  (%r)", b, loadargs)
+                logger.log(e_cmn.MIRE, "<< %r  (%r)", b, loadargs)
                 return msgpack.loads(b, **loadargs)
 
             def msgpackdumps(b):
-                logger.debug(">> %r  (%r)", b, dumpargs)
+                logger.log(e_cmn.MIRE, ">> %r  (%r)", b, dumpargs)
```

Comment:
i value this log message.  it may be 10 years old, but 10 years ago these two log messages was vital to troubleshooting cobra.  that's why i made it MIRE here.  i don't ever want it to trigger unless we're expecting the glut.

Reply:
Okay, we can have a log message there, that's fine. I'm just not a fan of slamming a slew of raw bytes to the console. I've found it's not a quick or useful workflow and ends up diluting other more useful messages.

But I also really really don't like our custom log levels. EMULOG is fine, but the others are a mess that I would like to not lean into until we can bring some actual structure and discipline to our logging. I find our stream of conscious log writing to be more of a hindrance when I'm debugging than anything else.

But that's a conversation for a different day. For now that, maybe we can revert this line to just being a debug and have that knock down drag out fight later?